test/cases/ostree-simplified-installer.sh

```diff
@@ -173,8 +186,15 @@ if [[ "$FDO_USER_ONBOARDING" == "true" ]]; then
 fdouser ALL=(ALL) NOPASSWD: ALL
 EOF
     sudo /usr/local/bin/yq -iy '.service_info.files |= [{path: "/etc/sudoers.d/fdouser", source_path: "/tmp/fdouser"}]' /etc/fdo/aio/configs/serviceinfo_api_server.yml
-    sudo systemctl restart fdo-aio
 fi
+
+# Fedora iot-simplified-installer uses /dev/vda3, https://github.com/osbuild/osbuild-composer/issues/3527
```

Comment:
Could you please move this part into line 28?

Reply:
Thanks, fixed.